platforms/hyperledger-indy/configuration/roles/setup/identity/tasks/nested_main.yaml

```diff
@@ -0,0 +1,119 @@
+- name: Select Admin Identity for Organisation {{ component_name }}
+  shell: |
+  
+    selectedAdmin=""
+    first_global_trustee=""
```

Comment:
rename the variable to e.g. first_global_admin as you use it for trustees + stewards